### DISCUSSION TITLE
How to install mutiple python versions?

### DISCUSSION THREAD

**Discussion by samisalamiws** [containerbase/base#245] [Q&A] [answered]
I'm trying to install multiple python versions using the following commands:
```
ARG PYTHON_VERSION=3.6.15
RUN install-tool python

ARG PYTHON_VERSION=2.7.18
RUN install-tool python

ARG PYTHON_VERSION=3.7.12
RUN install-tool python

ARG PYTHON_VERSION=3.8.12
RUN install-tool python
```

eventually, I can see all these versions installed in the image under `/usr/local/python`. but my problem is that I don't have the shortcuts of triggering `python3.7` and `python3.6` and `python2.7 `. only `python3.8` (the latest installed) is available 

I tried exporting the bin directory to the path also by using `RUN export PATH=$PATH:/usr/local/python/3.6.15/bin` but it keeps overriding it and I only get this path: `/home/dir1/.local/bin:/usr/local/python/3.8.12/bin:/home/dir1/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`

any ideas on how can I make this happen?

**Comment by rarkins** [comment]
@viceice is this supported yet, or could it be supported?

@samisalamiws is it your intention that the *last* python you install takes over `python` while the previous ones still are reachable using `python3.6`, `python3.7`, etc?

**Reply by samisalamiws** [reply]
the last python does take over, but the other versions are **not** reachable by simply running `python3.6`. If we want to use python3.6 we will need to trigger it directly from the folder `/usr/local/python/3.6.15/bin/python3.6`

**Reply by samisalamiws** [reply]
currently I found some workaround by running:
```
RUN cp /usr/local/python/${PYTHON_VERSION}/bin/python3.6 /usr/local/bin/
RUN cp /usr/local/python/${PYTHON_VERSION}/bin/python3 /usr/local/bin/
RUN cp /usr/local/python/${PYTHON_VERSION}/bin/pip3.6 /usr/local/bin/
```

**Comment by viceice** [comment, marked as answer]
This will be solved by #223

**Reply by samisalamiws** [reply]
thanks

**Reply by samisalamiws** [reply]
@viceice any ETA on this?

**Reply by viceice** [reply]
PR merged and released 🥳

**Reply by samisalamiws** [reply]
@viceice great job, thank you
